routes/auth.js

```diff
@@ -0,0 +1,191 @@
+// Import dependencies
+const smtpTransport = require('nodemailer-smtp-transport');
+const moment = require('moment');
+const config = require('../config/settings');
+const _      = require('lodash');
+const jwt    = require('jsonwebtoken');
+const authenticationHelpers = require('./authenticationHelpers');
+
+// create reset password token
+function createToken(user) {
+	return jwt.sign(_.omit(user, 'password'), config.secretKey, { expiresIn: 60*60*5 });
+}
+
+module.exports = function(router, passport, async, nodemailer, crypto, models){
+
+	const localUser = models.user_account;
+
+	router.post('/login', function(req, res, next) {
+		passport.authenticate('local-login', function(err, user, info) {
+			if (err) { 
+				return next(err); 
+			}
+			if (!user) {
+				return res.status(400).json(info); 
```

Comment:
is this the best HTTP code? 400 is bad request, and actually throws errors on frontend. is this the behaviour we'd like? typically in the case of mismatched username & password, the HTTP code is still 200, with proper error messages on the frontend. just have a look at google, facebook, twitter, github etc., invalid sign-ins doesn't trigger HTTP request errors